pkg/iptables/iptables.go

```diff
@@ -16,6 +16,7 @@ package iptables
 
 import (
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
Can you move this import down with the other 3rd party imports?

Reply:
done